packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -45,20 +45,18 @@ const BASE_CHAIN_ID = 'eip155'
 const MAX_PROVIDERS_COUNT = 100
 const TRANSACTION_CACHE_SIZE = 50
 const BLOCK_CACHE_SIZE = 50
-const V0_PROOF_TYPE = 'raw'
 const V1_PROOF_TYPE = 'f(bytes32)' // See: https://namespaces.chainagnostic.org/eip155/caip168
 
 const ABI = ['function anchorDagCbor(bytes32)']
 
 const iface = new Interface(ABI)
 
-//TODO (NET-1659): Finalize block numbers and smart contract addresses once CAS is creating smart contract anchors
 const BLOCK_THRESHHOLDS = {
-  'eip155:1': 1000000000, //mainnet
+  'eip155:1': 16688195, //mainnet
   'eip155:3': 1000000000, //ropsten
-  'eip155:5': 1000000000, //goerli
-  'eip155:100': 1000000000, //gnosis
-  'eip155:1337': 1000000, //ganache
+  'eip155:5': 8498671, //goerli
+  'eip155:100': 26509835, //gnosis
+  'eip155:1337': 1, //ganache
```

Comment:
This will mean that it is no longer possible to perform the old-style anchors with Ganache. Anyone testing with Ganache will need to deploy the anchor contract to it and use smart-contract style anchoring.  This makes tests more realistic but could add friction to people setting up test deployments using the `local` network with ganache.  FYI @oed let me know if you think I should hold off on this change for now.

@ukstv @stephhuynh18 please let me know if this will cause problems for the unit tests in the CAS codebase.

Reply:
@nathanielc would this add friction for you in your Keramik test setups?

Reply:
It will add friction,but if we can document the new process then we can add it as a step in keramik and then keep automation will solve it. 

Is adding the smart-contact to Ganache just a script I could run?

Reply:
I guess, it is all right from the CAS perspective. Means, we would remove some tests, and some code, and AFAIK that's it.

Reply:
If no one's worried about this change can I get some approvals on the PR?